website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx

```diff
@@ -27,7 +27,7 @@ global:
   image: 'hashicorp/consul-enterprise:1.4.3-ent'
 ```
 
-Add the name and key of the secret you just created to `server.enterpriseLicense`.
+Add the name and key of the secret you just created to `server.enterpriseLicense`, if using Consul version 1.10+.
```

Comment:
I'd change the `global.image` below to be `1.10` to make that clear.

Reply:
Good catch!